packages/fxa-settings/src/styles/UnitRow.scss

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.unit-row {
+  @apply flex flex-wrap px-4 pt-7 pb-8;
+
+  &-header {
+    @apply w-full mb-1;
+  }
+
+  &-header-value {
+    @apply w-full;
+  }
+
+  &-cta-container {
+    @apply w-full;
+  }
+
+  &-hr {
+    @apply mx-5 border-grey-100;
+  }
+}
+
+@screen mobileLandscape {
+  .unit-row {
+    @apply flex-no-wrap px-6;
+
+    &-header {
+      @apply flex-1 mb-0 mr-2;
+    }
+
+    &-header-value {
+      @apply flex-2;
+    }
+
+    &-cta-container {
+      @apply flex-1 flex justify-end;
+    }
+  }
+}
+
+@screen desktop {
+  .unit-row-header-value {
+    @apply flex-4;
+  }
+}
```

Comment:
I think I know why you named it this way, but how would you feel about calling this file `unit-row.scss`, so we're consistent? It could go in a `components` dir if you want to make the distinction?

Reply:
That's fine by me👍